fe/fe-core/src/main/java/com/starrocks/journal/bdbje/BDBJEJournal.java

```diff
@@ -64,10 +65,20 @@ public class BDBJEJournal implements Journal {
     public BDBJEJournal(BDBEnvironment bdbEnvironment, CloseSafeDatabase currentJournalDB) {
         this.bdbEnvironment = bdbEnvironment;
         this.currentJournalDB = currentJournalDB;
+        this.prefix = "";
     }
 
     public BDBJEJournal(BDBEnvironment bdbEnvironment) {
+        this(bdbEnvironment, "" /* prefix */);
+    }
+
+    public BDBJEJournal(BDBEnvironment bdbEnvironment, String prefix) {
         this.bdbEnvironment = bdbEnvironment;
+        this.prefix = prefix;
```

Comment:
I suggest you check if prefix is a number here.

Reply:
+1, or restrict `prefix` ending with `_`.

Reply:
done